.github/workflows/web-bff-ci.yml

```diff
@@ -0,0 +1,38 @@
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'web-bff/**' # Trigger only for changes in the web-bff directory
+  pull_request:
+    branches:
+      - main
+      - '**' # Trigger for all PR branches
+    paths:
+      - 'web-bff/**' # Trigger only for changes in the web-bff directory
+
+jobs:
+  web-bff:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write # Grant write permissions for packages
+      contents: read  # Ensure the workflow can read repository contents
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Log in to GitHub Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build Docker image
+        working-directory: web-bff
+        run: |
+          docker build -t ghcr.io/${{ github.repository_owner }}/web-bff:latest -f Dockerfile .
```

Comment:
why do you need to publish the docker image to the public GH container registry?